soc/riscv/riscv-privilege/sifive-freedom/soc.h

```diff
@@ -23,7 +23,7 @@
 #define SIFIVE_PINMUX_PINS            32
 
 /* Clock controller. */
-#define PRCI_BASE_ADDR               0x10008000
+#define PRCI_BASE_ADDR           0x10008000
```

Comment:
looks like this is accidental change, please remove it

Reply:
@snematbakhsh please send a separate PR to fix this.

Reply:
Sorry, it's my mistake. I removed those spaces to align with the definitions below, but in fact the alignment was already fine with the definitions above (and definitions below use tabs, so I definitely made things worse). PR #44560 sent.